### PR TITLE
Bipu

### PR DESCRIPTION
**Pull request by bariul28**
please reply me back

### CODE REVIEW

File: src/main/java/pages/CheckBoxesPage.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class CheckBoxesPage {
+
+    private String loginURL = "http://the-internet.herokuapp.com/checkboxes";
```
Comment (ashraf1190) [inline src/main/java/pages/CheckBoxesPage.java]: this is not a login URL

File: src/main/java/pages/HomeScreen.java [hunk 2]
```diff
@@ -10,6 +10,9 @@
 
     private By formAuthlinkLocator = By.linkText("Form Authentication");
     private By forgotPasswordLocator = By.linkText("Forgot Password");
+    private By checkBoxlocator = By.linkText ("Checkboxes");
+    private By checkBoxoption1 = By.xpath ("//*[@id=\"checkboxes\"]/input[1]");
```
Comment (ashraf1190) [inline src/main/java/pages/HomeScreen.java]: this should be in CheckBoxesPage.java

File: src/main/java/pages/HomeScreen.java [hunk 3]
```diff
@@ -10,6 +10,9 @@
...
+    private By checkBoxoption2 = By.xpath("//*[@id=\"checkboxes\"]/input[2]");
```
Comment (ashraf1190) [inline src/main/java/pages/HomeScreen.java]: this should be in CheckBoxesPage.java

File: src/main/java/pages/HomeScreen.java [hunk 4]
```diff
@@ -25,4 +28,10 @@ public void clickFormAuth() {
     public void clickForgotPassword() {
         click(forgotPasswordLocator);
     }
+
+    public void clickCheckboxes() {click(checkBoxlocator);}
+
+    public void clickoption1() {click (checkBoxoption1);}
```
Comment (ashraf1190) [inline src/main/java/pages/HomeScreen.java]: this should be in CheckBoxesPage.java

File: src/main/java/pages/HomeScreen.java [hunk 5]
```diff
@@ -25,4 +28,10 @@ public void clickFormAuth() {
...
+
+    public void clickoption2() {click ( checkBoxoption2 );}
```
Comment (ashraf1190) [inline src/main/java/pages/HomeScreen.java]: this should be in CheckBoxesPage.java

**Review by ashraf1190 — CHANGES_REQUESTED** [review]
There are a few things missing in this test 

1. How do we know that those options are selected? You can use a method to verify if the checkboxes are selected.
2. The CheckBoxesPage.java is basically empty? WHY?
3. Verify title on CheckBoxesPage so you know youre on the correct page

File: src/main/java/pages/CheckBoxesPage.java (hunk 1 above)
Comment (bariul28) [reply src/main/java/pages/CheckBoxesPage.java]: ok i will. and send you to review

File: src/main/java/pages/HomeScreen.java (hunk 2 above)
Comment (bariul28) [reply src/main/java/pages/HomeScreen.java]: ok

File: src/main/java/pages/HomeScreen.java (hunk 4 above)
Comment (bariul28) [reply src/main/java/pages/HomeScreen.java]: ok

**Comment by bariul28** [comment]
thanks for your valuable review.

**Review by bariul28 — COMMENTED** [review]
please review my work @ashraf1190

**Review by ashraf1190 — APPROVED** [review]

**Comment by ashraf1190** [comment]
Everything looks good I will merge your changes

**Pull request merged**